packages/browser/src/feedback.ts

```diff
@@ -5,7 +5,7 @@ import {
 } from '@sentry-internal/feedback';
 import { lazyLoadIntegration } from './utils/lazyLoadIntegration';
 
-// The full feedback widget, with everything pre-loaded
+/** Add a widget to capture user feedback to your application. */
```

Comment:
There is a sibling file feedbackAsync that could probably also have the comment fixed up

Reply:
updated this as well!